presidio-analyzer/conf/transformers.yaml

```diff
@@ -0,0 +1,7 @@
+nlp_engine_name: transformers
+models:
+  -

```

Comment:
Need to fix this yaml.

Reply:
What's wrong with it? Checked with YAML validator and it's valid

Reply:
might be wrong but there is a line and the content one line after...

Reply:
bur if it's valid I'll resolve :)